src/cache.c

```diff
@@ -10,8 +10,8 @@
 #include "util.h"
 #include "banned.h"
 
-#define CACHE_LOCATION "~/.cache/jgmenu/icons"
-static struct sbuf *cache_location;
+#define DEFAULT_CACHE_LOCATION "~/.cache/jgmenu/icons"
+struct sbuf *cache_location;
```

Comment:
I prefer keeping `static struct` here and suggest adding a `char * cache_get_dir()` rather than adding an `extern` in `src/icon.c`.